shiny/experimental/ui/_output.py

```diff
@@ -0,0 +1,280 @@
+    click
+        This can be a boolean or an object created by :func:`~shiny.ui.click_opts`. The
+        default is `False`, but if you use `True` (or equivalently, `click_opts()`), the
+        plot will send coordinates to the server whenever it is clicked, and the value
+        will be accessible via `input.xx_click()`, where `xx` is replaced with the ID of
+        this plot. The input value will be a dictionary with `x` and `y` elements
+        indicating the mouse position.
+    dblclick
+        This is just like the `click` parameter, but for double-click events.
+    hover
+        Similar to the `click` argument, this can be a boolean or an object created by
+        :func:`~shiny.ui.hover_opts`. The default is `False`, but if you use `True` (or
+        equivalently, `hover_opts()`), the plot will send coordinates to the server
+        whenever it is clicked, and the value will be accessible via `input.xx_hover()`,
+        where `xx` is replaced with the ID of this plot. The input value will be a
+        dictionary with `x` and `y` elements indicating the mouse position. To control
+        the hover time or hover delay type, use :func:`~shiny.ui.hover_opts`.
+    brush
+        Similar to the `click` argument, this can be a boolean or an object created by
+        :func:`~shiny.ui.brush_opts`. The default is `False`, but if you use `True` (or
+        equivalently, `brush_opts()`), the plot will allow the user to "brush" in the
+        plotting area, and will send information about the brushed area to the server,
+        and the value will be accessible via `input.plot_brush()`. Brushing means that
+        the user will be able to draw a rectangle in the plotting area and drag it
+        around. The value will be a named list with `xmin`, `xmax`, `ymin`, and `ymax`
+        elements indicating the brush area. To control the brush behavior, use
+        :func:`~shiny.ui.brush_opts`. Multiple `output_image`/`output_plot` calls may
+        share the same `id` value; brushing one image or plot will cause any other
+        brushes with the same `id` to disappear.
+
+    Returns
+    -------
+    :
+        A UI element
+
+    See Also
+    -------
+    ~shiny.render.plot ~shiny.ui.output_image
+    """
+
+    # NEW
+    if fill == "!inline":
+        fill = not inline
+    # /NEW
+
+    res = output_image(
+        id=id,
+        width=width,
+        height=height,
+        inline=inline,
+        click=click,
+        dblclick=dblclick,
+        hover=hover,
+        brush=brush,
+        # NEW
+        fill=fill,
+        # /NEW
+    )
+    res.add_class("shiny-plot-output")
+    return res
+
+
+@add_example()
+def output_image(
+    id: str,
+    width: str = "100%",
+    height: str = "400px",
+    *,
+    inline: bool = False,
+    click: bool | ClickOpts = False,
+    dblclick: bool | DblClickOpts = False,
+    hover: bool | HoverOpts = False,
+    brush: bool | BrushOpts = False,
+    # NEW
+    fill: bool = False,
```

Comment:
This should have the same interface as in `output_plot`.

Also make a note about this in bslib.

Reply:
Asked in https://github.com/rstudio/shiny/issues/3813

Reply:
Resolving here as an issue as been made